File: upsideCakes/Models/Pedido.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace upsideCakes.Models
{
    public class Pedido
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int id { get; set; }
        public DateOnly dataCriacao { get; set; }

        public int funcionarioID { get; set; }

        public int gerenteID { get; set; }

        public int qtde {  get; set; }

        [NotMapped]
        public List<Produto>? itens { get; set; }
    }
}

File: upsideCakes/AppUpsideCakes/src/app/models/Pedido.ts
import { Produto } from "./Produto"
 
export class Pedido {
    id: Number = 0;
    dataCriacao: Date = new Date('2023-11-11');
    funcionarioID: Number = 0;
    gerenteID: Number = 0;
    produto: Produto = new Produto();
    qtde: Number = 0;
}
